File: Example Code/02 Branches/04 Nested Statements/BeerAndAge.py
wantBeer = raw_input( "Do you want a beer? (y/n): " )

if ( wantBeer == "y" or wantBeer == "Y" ):
	
	age = raw_input( "What is your age? " )
	
	if ( age < 21 ):
		print( "You can't have a beer!" )
		
	else:
		print( "Here is a beer!" )
		
elif ( wantBeer == "n" or wantBeer == "N" ):
	print( "OK, here is a soda instead." )

else:
	print( "I don't know what '" + wantBeer + "' means!" )

File: Example Code/02 Branches/04 Nested Statements/BeerAndAge.cpp
#include <iostream>
using namespace std;

int main()
{
    cout << "Do you want a beer? (y/n): ";
    char wantBeer;
    cin >> wantBeer;

    if ( wantBeer == 'y' || wantBeer == 'Y' )
    {
        cout << "What is your age? ";
        int age;
        cin >> age;

        if ( age < 21 )
        {
            cout << "You can't have a beer!" << endl;
        }
        else
        {
            cout << "Here is a beer!" << endl;
        }
    }
    else if ( wantBeer == 'n' || wantBeer == 'N' )
    {
        cout << "OK, here is some soda instead." << endl;
    }
    else
    {
        cout << "I don't know what '" << wantBeer << "' means!" << endl;
    }

    return 0;
}
